lib/jormungandr/test/integration/Main.hs

```diff
@@ -99,6 +100,7 @@ main = withLogging Nothing Info $ \logging -> do
             describe "Mnemonics CLI tests" $ parallel (MnemonicsCLI.spec @t)
             describe "Miscellaneous CLI tests" $ parallel (MiscellaneousCLI.spec @t)
             describe "Launcher CLI tests" $ parallel (LauncherCLI.spec @t)
+            describe "Stake Pool Metrics" MetricsSpec.spec
```

Comment:
Haven't yet look into it but, why is that an integration test (and not a unit test)?

Reply:
To avoid having to mock `JormungandrClient`.

Reply:
Sorry, I hadn't noticed the Jormungandr import!